lib/core/userinfo/services/ecm-user.service.ts

```diff
@@ -42,7 +42,9 @@ export class EcmUserService {
     getUserInfo(userName: string): Observable<EcmUserModel> {
         return from(this.apiService.getInstance().core.peopleApi.getPerson(userName))
             .pipe(
-                map(data => <EcmUserModel> data['entry']),
+                map((data) => {
+                    return new EcmUserModel(data['entry']);
```

Comment:
why do data['entry'] and not just data.entry? and data which kind of type is it?

Reply:
the return type is an Object  '{}'
if we use data.entry, it leads to this error: Property 'entry' does not exist on type {}

Reply:
Dot notation is much easier to read than bracket notation I would  prefer to use it as in the rest of the code.   Property 'entry' does not exist on type {} means that data is empty maybe we need to check for it. I guess the type of data should be PersonEntry and please change the name of the variable data from 

```
(data)  to (personEntry:PersonEntry)
```

Reply:
so this 
`new EcmUserModel(data['entry']);`
will become 
`new EcmUserModel(person.entry]);`